contrib/test/integration/system-packages.yml

```diff
@@ -5,11 +5,12 @@
 - name: Make sure we have all required packages
   yum:
     state: present
+    enablerepo: powertools
     name:
-      - atomic-registries
+      #- atomic-registries
       - container-selinux
       - curl
-      - device-mapper-devel
+      #- device-mapper-devel
```

Comment:
Let's uncomment the packages so jenkins keeps working.

Reply:
if we uncomment though the prow will fail :thinking: